lib/rules/require-toggle-inside-transition.js

```diff
@@ -47,10 +56,6 @@ module.exports = {
     return utils.defineTemplateBodyVisitor(context, {
       /** @param {VElement} node */
       "VElement[name='transition'] > VElement"(node) {
-        const child = node.parent.children.find(utils.isVElement)
-        if (child !== node) {
-          return
-        }
```

Comment:
I don't think this change is necessary. Could you please revert?

Reply:
If this condition is kept, the following case would be failed.
```html
<template>
  <Transition appear>
    <div v-if="show" />                           // valid
    <div />                                                 // should be marked as error, but not
  </Transition>
</template>
```

Reply:
I think that second `<div>` is invalid for another reason.
If you want to check whether the elements within that `<transition>` are valid, I think we need to add another rule.

Reply:
[demo](https://play.vuejs.org/#eNp9kb1uwjAUhV/lygutRMnAhgJSWzG0Q1u1jF5McgMGx7b8E5BQ3r3XpoUMCE/2+c6xzrVP7NnaSReRzVjpKydtAI8h2gXXsrXGBTiBwwZ6aJxpYUTWEdeV0Z6MW3OAecIPjVAeH7kui/MlFKdDwNYqEZBOAOXKCe1lkEaDsBaFyzKBWnbQPclmzlm6kjMoFgnkVRTQCSXrgXdAh4ucFI+qhjVCK9weaxAe0DnjxrCOAbQJuUlxrUIdyuLSk41Z8DRcIzeTnTeaXuWUEpxVprVSofu0KeU5m0EmiQmlzOE9a8FFHP/r1Rar/Q19549J4+zLoUfXIWcXFoTbYDjj5c8HHml/ga2poyL3HfiN3qiYOp5tL1HXVHvgy23f8t9KvVn55TEgPcbfUKlocvbZzxn99+ud0a91p5NpznHds/4XUzm/5w==)

Reply:
Okay, that make sense 🤔